docs/DeveloperGuide.md

```diff
@@ -457,29 +457,12 @@ borrowed books tracking, and module-related functions. This increase users' effi
 
 ## Appendix: Instructions for manual testing
 Below are the steps required for manual testing of termiNus
-
-### Initial launch and shutdown
-
-1. Initial Launch
-    a. Download the latest version of `termiNus` from [here](https://github.com/AY2021S1-CS2113-T14-3/tp/releases/latest) and copy the jar file to a new folder.
-
-    b. Launch termiNus by typing `java -jar termiNus.jar` and press enter.
-
-    c. To get a detailed description, refer to [User Guide](UserGuide.md).
-
-
-2. Shutdown 
-    a. Once testing is conducted, exit the program by entering `bye`.
-
-### Features and functions
-
 <!-- @@author Cao-Zeyu -->
-## Instructions for manual testing
-
 ### Launch and shutdown
 1. Initial launch
-    1. Download the jar file and copy to an empty folder.
-    2. Open a command line window in the same directory and type `java -jar termiNus.jar` to launch.
+    1. Download the latest version of `termiNus` from [here](https://github.com/AY2021S1-CS2113-T14-3/tp/releases/latest) 
+    and copy the jar file to an empty folder.
+    2. Open a command line window in the same directory and launch termiNus by typing `java -jar termiNus.jar` and press enter.
 2. Shutdown
```

Comment:
For this part, u can modify the pointers to this : 
![image](https://user-images.githubusercontent.com/42372471/98467470-76a0c180-2210-11eb-9433-4baf2d07e59e.png)

Reply:
Do include the <br> if not they will be all on the same line

Reply:
Sure!